src/api/app/assets/javascripts/webui/application/kiwi_editor.js

```diff
@@ -117,6 +123,34 @@ function closeDescriptionDialog() {
 }
 
 
+function closePreferencesDialog() {
+  var fields = $(this).parents('.nested-fields');
+  var dialog = fields.find('.dialog');
+
+  var elements = fields.find('.fill');
+  for(var i=0; i < elements.length; i++) {
+    var object = dialog.find("[id$='" + $(elements[i]).data('tag') + "']");
+    if ( object.val() != "") {
```

Comment:
I don't think we have a style guide for this, and I just read that [mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators) is actually advising to use the `standard equality operators (== and !=)` by default. Which surprises me, since I've been taught many times to not use them due to their surprising results, eg. ` 0 == "" ` would evaluate to true.

That said, I would prefer to use the strict equality operators (!== / ===). But it's a nitpick.